.github/actions/phpunit/action.yml

```
@@ -0,0 +1,60 @@
+name: 'Run PHPUnit and Behat tests'
+
+description: 'Sets up PHP and its dependencies, installs WordPress test library along with plugins and themes then runs PHPUnit and Behat tests.'
+
+inputs:
+    php-version:
+      description: 'PHP version to use.'
+      required: true
+      type: string
+    wordpress-version:
+      description: 'WordPress version to use. Accepts a numeric string of the version, "nightly", or "latest".'
+      required: true
+      type: string
+    allow-failure:
+      description: 'Whether or not the job is allowed to fail. Defaults to false.'
+      required: false
+      type: boolean
+      default: false
```

Indented with 4 and 6 spaces instead of 2 and 4.
```suggestion
  php-version:
    description: 'PHP version to use.'
    required: true
    type: string
  wordpress-version:
    description: 'WordPress version to use. Accepts a numeric string of the version, "nightly", or "latest".'
    required: true
    type: string
  allow-failure:
    description: 'Whether or not the job is allowed to fail. Defaults to false.'
    required: false
    type: boolean
    default: false
```